### PR TITLE
Fix `ActionController::Parameters#==` bug

### PR DESCRIPTION
**Pull request by maclover7**
See bug #21032.

cc @sikachu

### CODE REVIEW

**Comment by maclover7** [comment]
:heart:

**Comment by sikachu** [comment]
Thanks!

**Comment by maclover7** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: actionpack/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+*   Fix `ActionController::Parameters==`
+
```
Comment (vipulnsward) [inline actionpack/CHANGELOG.md]: probably needs expansion?

Comment (kaspth) [inline actionpack/CHANGELOG.md]: I'd say we just remove it altogether. It's fixing the new unreleased parameters, not the one from 4.2.

Comment (vipulnsward) [inline actionpack/CHANGELOG.md]: True. Done here https://github.com/rails/rails/pull/23178

**Pull request merged**